logstash/tests/logstash_test.py

```diff
@@ -571,3 +571,22 @@ def test_adding_a_service():
     assert len(s['spec']['ports']) == 1
     assert s['spec']['ports'][0] == {
         'name': 'beats', 'port': 5044, 'protocol': 'TCP', 'targetPort': 5044}
+
+def test_setting_fullnameOverride():
+    config = '''
+fullnameOverride: 'logstash-custom'
+'''
+    r = helm_template(config)
+
+    custom_name = 'logstash-custom'
+    assert custom_name in r['daemonset']
```

Comment:
Logstash is using a statefulset and not a daemonset.

Reply:
Fixed. I did not attempt to run the tests locally thanks for triggering the Jenkins build.